tools/StaticAnalysis/Exceptions/MissingAssemblies.csv

```diff
@@ -19,3 +19,8 @@
 "src\Package\Debug\ServiceManagement\Azure\Sql","Microsoft.Web.Delegation","7.1.0.0","Microsoft.Web.Deployment","0","3000","Missing assembly Microsoft.Web.Delegation referenced from Microsoft.Web.Deployment","Ensure that the assembly is included in the Wix file or directory"
 "src\Package\Debug\ServiceManagement\Azure\Sql","Microsoft.Web.Administration","7.0.0.0","Microsoft.Web.Deployment","0","3000","Missing assembly Microsoft.Web.Administration referenced from Microsoft.Web.Deployment","Ensure that the assembly is included in the Wix file or directory"
 "src\Package\Debug\ServiceManagement\Azure\Sql","Microsoft.Web.Deployment.Tracing","9.0.0.0","Microsoft.Web.Deployment","0","3000","Missing assembly Microsoft.Web.Deployment.Tracing referenced from Microsoft.Web.Deployment","Ensure that the assembly is included in the Wix file or directory"
+"D:\workspace\powershell-sign\src\Package\Release\ServiceManagement\Azure\Compute","Microsoft.Web.Administration","7.0.0.0","Microsoft.Web.Delegation","0","3000","Missing assembly Microsoft.Web.Administration referenced from Microsoft.Web.Delegation","Ensure that the assembly is included in the Wix file or directory"
```

Comment:
@markcowl you might have to change this back to relative path or get a new pull once PR https://github.com/Azure/azure-powershell/pull/3462/ is merged.